persistence.js

```diff
@@ -525,9 +525,13 @@ function returnSubsForClient (subs) {
   }
 
   for (const subKey of subKeys) {
+    const sub = JSON.parse(subs[subKey])
     toReturn.push({
       topic: subKey,
-      qos: parseInt(subs[subKey])
+      qos: sub.qos,
+      rh: sub.rh,
+      rap: sub.rap,
+      nl: sub.nl
```

Comment:
What about making this more generic like what we do in other persistences?

Reply:
Good point, again I tried to stay as close to the original code as possible but it does make sense to make it more generic :-)

Reply:
If I am not mistaken, qlobber is not generic either so genericity will not help here unless qlobber (incl. the native part) supports generic subscription content. See e.g.: https://github.com/davedoesdev/qlobber/commit/376261b3de4b19372dcc4acd95fe4054f4fb34ff

That's why I on purpose did not make it generic otherwise some nasty bugs could occur due to qlobber.